packages/node/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "@envy/node",
   "version": "1.0.0",
   "description": "Node.js Network & Telemetry Viewer",
-  "main": "dist/index.js",
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",

```

Comment:
This was changed to allow for internal package imports - If this has negative implications I can revert this, just allowed for cleaner `import` in the example apps

Reply:
I think ultimately we'll want it to be `dist/index.js`, since usage will be against the built, published code rather than source.

Reply:
Gotcha, looks like it works without this change, reverted